contracts/BandExampleConsumerToken.cdc

```diff
@@ -11,20 +11,15 @@ by anyone based on payment provided in a FLOW token vault. The price of
 to enforce a minimum and calculate the cost in FLOW to mint the requested amount of 
 tokens.
 
-This simple example highlights trade-offs to consider regarding timing and 
-synchronization when requesting price quotes, particularly when integrating the 
-oracle from a dapp contract. It uses a naive approach to synchronously request the 
-quote from `BandOracle` within the function. This ensures the most accurate and up 
-to date price. It's also the most expensive. Especially because it's a spam attack 
-vector by which a malicious caller's DOS attack could quickly drain funds from the 
-dapp contract due to the synchronous implementation. Accessing the oracle price quote 
-directly in the code of a transaction rather than through contract code does is not 
-subject to the above spam risk, although cost may still be a consideration.
-
-An alternative approach is to use periodic price updates, enforcing asynchrony 
-through a cache for example. This mitigates the spam attack risk, however, it 
-increases exposure to market volatility. Applications must determine their level of 
-price accuracy vs cost according to their use cases and financial risk tolerance.
+This contract showcases a function enabling the exchange of Flow tokens for 
+`BandExampleConsumerTokens`. When the swap function is invoked, the contract 
+leverages the most recent FLOW/USD exchange rate stored.
```

Comment:
```suggestion
This example contract showcases a function enabling the exchange of Flow tokens for 
`BandExampleConsumerTokens`. When the swap function is invoked, the contract 
leverages the most recently stored FLOW/USD price quote for the token swap. 

It should be noted that this contract example is not representative of how oracle integration
would generally be implemented. The example highlights a use-case typically handled by 
Decentralized Exchanges (DEXs) which use oracle price quotes to perform token swaps. 
Unless you are operating an exchange, or otherwise exposing price quotes obtained to 
callers of your contract, other use-cases may only need to query current price quotes using 
a transaction.
```